torch/csrc/api/include/torch/nn/modules/containers.h

```diff
@@ -19,32 +19,32 @@ class ContainerListImpl : public CloneableModule<Derived> {
   }
 
   std::shared_ptr<Module> add(std::shared_ptr<Module> m) {
-    return append(m).children_.back();
+    return append(m).modules_.back();
```

Comment:
Should we change the name of this to `_modules_` so people won't be tempted to mess around with it?

Reply:
Honestly let's not care about this file, it will get removed once `Sequential` lands from https://github.com/pytorch/pytorch/pull/7508